src/Test.UnitTests.Sarif/WarningsTests.cs

```diff
@@ -24,7 +24,7 @@ public void Warnings_LogRuleWasExplicitlyDisabled()
 
             testLogger.Messages.Should().BeNull();
             testLogger.ToolNotifications.Should().BeNull();
-            testLogger.ConfigurationNotifications.Count.Should().Equals(1);
+            testLogger.ConfigurationNotifications.Count.Should().Be(1);
```

Comment:
>Be [](http://example.com/codeflow?start=65&length=2)

Heads up everyone, this rename will show up in other tests for sure.

Reply:
`.Count.Should().Equals() --> .Count.Should().Be()`
correct, I have noticed this before.